src/main/resources/fabric.mod.json

```
@@ -17,10 +17,10 @@
   },
   "mixins": ["carpet-addons-not-found.mixins.json"],
   "depends": {
-    "fabricloader": ">=0.14.22",
-    "minecraft": "1.20.2",
-    "java": ">=17",
-    "carpet": ">=1.4.112"
+    "fabricloader": ">=0.16.3",
+    "minecraft": ">=1.21.1",
```

```suggestion
    "minecraft": ">=1.20.6",
```

Copied this wrong, whoops

---

Resolved